staging/src/k8s.io/apiserver/pkg/server/options/etcd.go

```diff
@@ -396,7 +396,7 @@ func (f *StorageFactoryRestOptionsFactory) GetRESTOptions(resource schema.GroupR
 		EnableGarbageCollection:   f.Options.EnableGarbageCollection,
 		ResourcePrefix:            f.StorageFactory.ResourcePrefix(resource),
 		CountMetricPollPeriod:     f.Options.StorageConfig.CountMetricPollPeriod,
-		StorageObjectCountTracker: f.Options.StorageConfig.StorageObjectCountTracker,
+		StorageObjectCountTracker: storageConfig.StorageObjectCountTracker,
```

Comment:
I think we have to try both sources. Use `f.Options.StorageConfig.StorageObjectCountTracker` if it is set (it's more specific, and fall back on nil to `storageConfig.StorageObjectCountTracker` otherwise.